trunk/SUAVE/Input_Output/OpenVSP/get_fuel_tank_props.py

```diff
@@ -0,0 +1,178 @@
+## @ingroup Input_Output-OpenVSP
```

Comment:
rename this. The name props is unclear. Props could be propulsion, propeller, etc...